posts/fall2023/hey-future-interns/index.md

```diff
@@ -0,0 +1,97 @@
+
+> Jonathan: “My favorite was when we went to Sawtelle together. It was the butterfly season when you start to get to know everyone. It was my favorite because I was able to learn about people and get to know each individual. After that first impression, I thought that we had a good mix and everybody seemed to work well with each other.”
+
+\
+From the socials to the workshops to the hackathons, Hack definitely provides a unique space at UCLA. Whether you are a new freshman or have been in the college environment for years, we encourage you to check us out! If we really spark your interest, we encourage you to apply to our internship program! Who knows, maybe one day, your name will be on this website. With that, we’ll leave you with some of the final words that our interns had to say:
+	
+> Abigail: “Just apply and do it! You will not regret it. It’s very tight-knit and fun and you will feel that you’re doing something meaningful. You’re not only helping others learn but you’re helping yourself learn as well. [Hack] has a reputation for being close-knit and fun, and people from other ACM committees have asked me before how we do it. It’s really just the people, culture, and work that we get to do that makes it a great experience.”
+
+> Andy: “If you find yourself in Hack, then you’ve found yourself in a pretty solid place.”
+
+> Brooke: “Have a good time, don’t get too stressed about things.” 
+
+> Jenna: “Be chill, enjoy hosting the events and building things, and have fun!”
+
+> Jonathan: “Trust yourself and be yourself. Trust who you are, because in the end, we want who you are.”
+
+> Satyen: “Don’t take for granted the opportunities that you have at UCLA. At Hack, you’ll find that everyone is very friendly and you’ll learn a lot.”
+
+> Shiyu: “Join hack or I’ll hunt u down.”
+
+\
+As for ourselves, we have had similar experiences from when we were interns two years ago. Hack has taught us technical coding skills, organizational skills to direct and teach events, leadership skills, and given us a great community of friends. In fact, both of us (Nathan and James) are now roommates! So, there really is some truth to the theme of Hack helping you make some close homies. We aren’t gaslighting you, we swear.
+
+We hope that this blog post gives you a bit of insight into what it’s like at ACM Hack. Feel free to reach out, too! We’d love to hear from you. If you think Hack is the right place, definitely apply! We look forward to getting to know you!
+
+(By the way, if you’ve made it this far and are applying to Hack, write the phrase “pies are delicious yum yum yum” in the extra comments question. It’ll be fun for us to see who’s read this article. You can tell others, too, but it’ll make you less special lol. Otherwise, it’ll just be a lil easter egg between you and us.)
```

Comment:
Might be more clear to say 

"in the extra comments question of our internship application."